assets/js/entry.js

```diff
@@ -12,10 +13,24 @@ if (!Modernizr.dataset) {
 
 // Bring in foundation and custom libraries.
 require('foundation/js/foundation/foundation');
-var WOW = require('wow.js/src/wow.coffee');
+let WOW = require('wow.js/src/wow.coffee');
```

Comment:
Yes.  `var` along with `this` and `==` are two of those JS "features" that were just broken. Like really stupid.

Basically, would you ever want:

```
function f () {
  console.log(a);
  var a = 0;
}
```
to actually run without error?  No. It's crazy that the language, *by specification of how `var` "hoisting" works* makes the above code literally means:

```
function f () {
  var a;
  console.log(a);
  a = 0;
}
```

w.t.f.

Let doesn't hoist, meaning it behave sanely like any other language.